packages/office-ui-fabric-react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -38,9 +39,13 @@ export class SuggestionsItem<T> extends BaseComponent<ISuggestionItemProps<T>, {
           className
         )}
       >
-        <CommandButton onClick={onClick} className={css('ms-Suggestions-itemButton', styles.itemButton)}>
-          {RenderSuggestion(suggestionModel.item, this.props)}
-        </CommandButton>
+        {renderSuggestionsItemAsDiv ? (
+          <div className={css('ms-Suggestions-itemButton', styles.itemButton)}>{RenderSuggestion(suggestionModel.item, this.props)}</div>
```

Comment:
Should the containing item then have to handle to onclick?

Reply:
Also I think that we can make this more generic than just a boolean. Instead of renderSuggestionsItemAsDiv, why not make it a SuggestionsItemContainerAs, and then follow that model so that a consumer could pass through a span, or even another type of button.

Reply:
Perhaps using `IRenderFunction<T>`